README.md

```diff
@@ -30,18 +30,6 @@ npx install-peerdeps --dev @grafana/eslint-config
 
 If using a different package manager, run `npm info "@grafana/eslint-config@latest" peerDependencies` to list the peer dependencies and versions. Then run `yarn add --dev <dependency1>@<version> <dependency2>@<version> <dependency3>@<version>` for the listed peer dependencies.
 
-## Configuration (legacy: .eslintrc\*)
-
-Extend our configuration to get reasonable defaults:
-
-```json
-{
-  "extends": ["@grafana/eslint-config"]
-}
-```
-
-It will automatically handle `*.(js|ts|tsx)` files.
-
```

Comment:
Did you mean to delete this? This package works for both legacy and flat configs.